modules/cucumber-reportportal-formatter.js

```diff
@@ -50,6 +50,7 @@ const createRPFormatterClass = (config) => {
         typeof this.config.scenarioBasedStatistics === 'boolean'
           ? this.config.scenarioBasedStatistics
           : false;
+      this.skippedIssue = config.skippedIssue;
```

Comment:
Why are you duplicating this option as additional class field as it's just a reassignment without any calculations?
I suppose that `this.config.skippedIssue` can be used directly.